content/nb_collectives_intro.tex

```diff
@@ -0,0 +1,35 @@
+An \openshmem nonblocking collective operation, like blocking collective
+operation, is a group communication operation among the
+participants of the team. All participants of the team are required to call the
+collective operation.
+
+\begin{enumerate}
+
+\item Invocation semantics: Upon invocation of a collective routine interface,
+the operation is posted and returned immediately. All participants of the Team
+should call this routine.
+
+\item Collective Types: The nonblocking variants supported include alltoall,
+broadcast, and reduction collectives. Other collective operations such as
+collect, barrier, alltoalls, and sync will not have nonblocking variants. The
+reduction types supported are defined in Table \ref{teamreducetypes}.
+
+\item Completion semantics:  \openshmem programs can learn the status of the collective operations
+using the \FUNC{shmem\_req\_test} routine and can be completed using
+the \FUNC{shmem\_req\_wait} routine.
```

Comment:
I don't see the definition of `shmem_req_wait` in this PR. Am I missing it?

Reply:
Is it not written, yet.

Reply:
Fixed